PendingReleaseNotes

```diff
@@ -1,5 +1,9 @@
 >=16.0.0
 --------
+* The "global_init_postfork_start()", which is ususally called in the child
+  porcesses after the fork(), will always reexpand the config options meta
+  to make sure the ``$pid`` is correctly expanded.
```

Comment:
This is too technical. Let's adjust to:

````
* $pid expansion in config paths like `admin_socket` will now properly expand to the daemon pid for commands like `ceph-mds` or `ceph-osd`. Previously only `ceph-fuse`/`rbd-nbd` expanded `$pid` with the actual daemon pid.
````

Reply:
Looks better and fixed it. Thanks.